src/lib/routes/admin-api/context.ts

```diff
@@ -172,6 +188,9 @@ export class ContextController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Context'],
+                    summary: 'Validates a context field',
+                    description:
+                        'Used by the Unleash frontend to validate that the form for creating a context field is valid (e.g. that the name of the context field is URL safe)',
```

Comment:
I don't think we need to tell the user where we use this? But I'm not sure it matters much.

What does it do if it can't? Does it return anything about why the field can't be created? And is it returned as a 200 with a body or as a 400?

```suggestion
                        'Check whether the provided data can be used to create a context field. If the data is not valid, ...?',
```